database/factories/ModelFactory.php

```diff
@@ -452,5 +452,19 @@ function (Faker\Generator $faker, $attributes) {
             [
               'uuid' => $uuid
             ];
-    }
-);
+});
+
+// Delivery - a schedule of vouchers sent somewhere
+$factory->define(App\Delivery::class, function (Faker\Generator $faker, $attributes) {
+
+    $centre_id = (empty($attributes['centre_id']))
+        ? factory(App\Centre::class)->create()
+        : $attributes['centre_id']
+    ;
+
+    return [
+        'range' => '',
+        'dispatched_at' => Carbon::today(),
```

Comment:
If using today rather than now, maybe go back to using `dispatched_on` ?

Reply:
Needs changing for variable dates anyway